apisix/plugins/limit-count/init.lua

```diff
@@ -50,6 +50,9 @@ local policy_to_additional_properties = {
             redis_port = {
                 type = "integer", minimum = 1, default = 6379,
             },
+            redis_username = {
+                type = "string", minLength = 0,
```

Comment:
Does redis accept "" as `username`? or minLength = 1?

Reply:
Hi @monkeyDluffy6017, if there is  "" as username in the configuration, we don't use it (see plugins/limit-count/limit-count-redis.lua l.62). Redis accept minLength = 1.